styles/index.styles.js

```diff
@@ -67,4 +67,14 @@ const AppWrapper = styled.div`
   background-color: #dae0e6;
 `;
 
-export { AppWrapper, GlobalStyle };
+const Navigation = styled.a`
```

Comment:
You have this component in both the index and addCommunity, right? Why not create it as an actual component that has no functionality and only design. I mean, in the components folder like Tranding.

Reply:
Done!